python/interpret_community/explanation/explanation.py

```diff
@@ -352,9 +353,15 @@ def num_examples(self):
         :rtype: int
         """
         if ClassesMixin._does_quack(self):
-            return len(self.local_importance_values[0])
+            if sp.sparse.issparse(self.local_importance_values[0]):
```

Comment:
I don't think there's a better way but this is kinda gross

Reply:
I think I've made it a bit more uniform now, and also more memory efficient in dense case (using local_importance_values doubles memory usage as opposed to _local_importance_values)